src/pages/accounts/SignUpPage.tsx

```diff
@@ -95,93 +90,72 @@ function SignUpPage() {
-				</PassrLogo>
-
-				<InputContainer>
-					<InputLabel>
-						<label>Email *</label>
-					</InputLabel>
-					<InputBox
-						type="text"
-						value={email}
-						onChange={(e) => {
-							if (validEmail(e.target.value)) setEmailErr(false);
-							setEmail(e.target.value);
-						}}
-						onBlur={(e) => {
-							if (!validEmail(e.target.value)) setEmailErr(true);
-						}}
-					/>
-				</InputContainer>
-
-				<InputContainer>
-					<InputLabel>
-						<label>Confirm Email *</label>
-					</InputLabel>
-					<InputBox
-						type="text"
-						value={confirmEmail}
-						onChange={(e) => {
-							if (compareEmail(email, confirmEmail))
-								setEmailErr(false);
-							setConfirmEmail(e.target.value);
-						}}
-						onBlur={() => {
-							if (!compareEmail(email, confirmEmail))
-								setEmailErr(true);
-						}}
-					/>
-				</InputContainer>
-
-				<InputContainer>
-					<InputLabel>
-						<label className="Password-label">Password *</label>
-					</InputLabel>
-					<InputBox
-						type="password"
-						value={userPass}
-						onChange={(e) => {
-							if (validPass(e.target.value)) setPassErr(false);
-							setUserPass(e.target.value);
-						}}
-					/>
-				</InputContainer>
-
-				<InputContainer>
-					<InputLabel>
-						<label className="Password-label">
-							Confirm Password *
-						</label>
-					</InputLabel>
-					<InputBox
-						type="password"
-						value={confirmUserPass}
-						onChange={(e) => {
-							if (comparePass(userPass, confirmUserPass))
-								setPassErr(false);
-							setConfirmUserPass(e.target.value);
-						}}
-					/>
-				</InputContainer>
-
-				<Links>
-					<a
-						href="https://reactjs.org"
-						target="_blank"
-						rel="noopener noreferrer"
-					>
-						Already have an account? Log in here.
-					</a>
-				</Links>
-				<SignUpButton
-					type="button"
-					disabled={emailErr || passErr}
-					onClick={onSubmit}
-				>
+				<PassrLogo width="300px" height="300px" />
+
+				<TextInput
+					label="Email"
+					required
+					placeholder={'example@example.com'}
+					type="text"
+					value={email}
+					onChange={(e) => {
+						if (validEmail(e.target.value)) setEmailErr(false);
+						setEmail(e.target.value);
+					}}
+					onBlur={(e) => {
+						if (!validEmail(e.target.value)) setEmailErr(true);
+					}}
+				/>
+				<TextInput
+					label="Confirm Email"
+					required
+					placeholder="example@example.com"
+					type="text"
+					value={confirmEmail}
+					onChange={(e) => {
+						if (compareEmail(email, confirmEmail))
+							setEmailErr(false);
+						setConfirmEmail(e.target.value);
+					}}
+					onBlur={() => {
+						if (!compareEmail(email, confirmEmail))
+							setEmailErr(true);
+					}}
+				/>
+				<TextInput
+					label="Password"
+					required
+					type="password"
+					value={userPass}
+					onChange={(e) => {
+						if (validPass(e.target.value)) setPassErr(false);
+						setUserPass(e.target.value);
```

Comment:
Is the password being blurred somehow when being inputted (ex. turning the characters into *** or dots or something)? If not should this be a feature?

Reply:
So there is a special input type password that is native html that will * out the password (`<input type="password />`). As you can see I pass through the type to the input element in the text input.